components/patient-data-sharing/push-api/src/test/java/org/phenotips/data/push/internal/DefaultPushServerResponseTest.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see http://www.gnu.org/licenses/
+ */
+package org.phenotips.data.push.internal;
+
+import net.sf.json.JSONObject;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class DefaultPushServerResponseTest {
+
+    private DefaultPushServerResponse defaultPushServerResponse;
+    private JSONObject response;
+
+    @Before
+    public void setUp() {
+        response = new JSONObject();
+        defaultPushServerResponse = new DefaultPushServerConfigurationResponse(response);
+    }
+
+    @Test
+    public void generateIncorrectCredentialsJSONTest() {
```

Comment:
Don't use `test` in method names, but a description of what you're testing, in general the name of the tested method and what behavior you're testing: `generateIncorrectCredentialsJSONWorksCorrectly`